observations/README.md

```diff
@@ -0,0 +1,1279 @@
+# Observation Mapping
+
+Main section links:
+- **[XML HL7 > JSON FHIR](#XML-HL7->-JSON-FHIR)**
+- **[JSON FHIR > XML HL7](#JSON-FHIR->-XML-HL7)**
```

Comment:
When clicking on this link it doesn't seem to do anything or take me to that sub-section.

Reply:
Thanks, changed the anchor links to GitHub format